### PR TITLE
CH-174831358-Integrate-Linter

### PR DESCRIPTION
**Pull request by developerbello**
## Description

Ticket - [174831358](https://www.pivotaltracker.com/n/projects/2466055/stories/174831358)

-  Configured ESLint with `.eslintrc.json` file (https://eslint.org/).
-  Added an executable script to package.json named `lint`.
-  I ensure that codes in `app` & `api` are `linted`.

## Additional Changes
- Installed Eslint npm package using `yarn add eslint` .
- To enable linters features on project use `yarn lint`.

### CODE REVIEW

**Comment by abdulfataiaka** [comment]
Please follow the requirements on the Ticket to complete expectations, thanks

**Comment by developerbello** [comment]
> Please follow the requirements on the Ticket to complete expectations, thanks

Ok, I will ensure to follow instructions based on the requirements. Thanks

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  ],
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+  "env": {
+    "browser": true,
+    "node": true,
+    "jest": true,
+    "mongo": true
```
Comment (abdulfataiaka) [inline .eslintrc.json:16]: Remove mongo and browser env since we are not running the code on the browser

File: package.json [hunk 2]
```diff
@@ -5,7 +5,13 @@
   "repository": "https://github.com/leickon/data-api",
   "author": "abdulfataiaka <abdulfataiaka@gmail.com>",
   "license": "MIT",
+  "scripts": {
+    "eslint": "esw --watch app api"
```
Comment (abdulfataiaka) [inline package.json:9]: We do not need to watch for changes as that might eat up the memory. We should only be able to run eslint to see errors or warning whenever we need to. `yarn eslint`

Also, we can configure VSCode to do the watching for us

**Review by abdulfataiaka — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (abdulfataiaka) [reply package.json:9]: Rename the script `eslint` to `lint`

File: .eslintrc.json (hunk 1 above)
Comment (developerbello) [reply .eslintrc.json]: Added changes as required

File: package.json (hunk 2 above)
Comment (developerbello) [reply package.json]: Added changes as required

**Review by abdulfataiaka — APPROVED** [review]

**Pull request merged**